ExposureApp/CRExposureNotification/ViewControllers/TabbarController.swift

```diff
@@ -10,12 +10,21 @@ class TabbarController: UITabBarController {
     }
     
     public func languageChanged(){
-        if let controller = self.viewControllers?[0]{
+        if let controller = self.viewControllers?.first {
             controller.title = "TabbarController.Exposures".localized()
         }
         
-        if let controller = self.viewControllers?[1]{
+        if let controller = self.viewControllers?.elementOrNil(at: 1) {
             controller.title = "TabbarController.NotifyOthers".localized()
         }
     }
 }
+
+public extension Array {
```

Comment:
I added this extension here but if you have a better place, folder with other extension, feel free to move it there.